transcoder.py

```diff
@@ -457,7 +457,7 @@ def copyCoverFiles(lossyTree):
   from shutil import copyfile # Use copyfile b/c this will *not* copy rights (which is error prone on gvfs/samba)
   from math import trunc 
   
-  log('Copy cover files to lossy tree: ' + lossyTree)
+  log('Embed cover-art to transcoded files: ')
```

Comment:
Value of lossyTree is ommittedm so is nit clear which lossy tree (ogg/mp3) is traversed

Reply:
Embed cover-art to transcoded files in lossyTree

Reply:
Embed cover-art to transcoded lossyTree files
Where lossyTree = MP3 or OGG